src/test/java/seedu/findvisor/logic/commands/EditCommandTest.java

```diff
@@ -146,6 +147,26 @@ public void execute_invalidPersonIndexFilteredList_failure() {
         assertCommandFailure(editCommand, model, Messages.MESSAGE_INVALID_PERSON_DISPLAYED_INDEX);
     }
 
+    /**
+     * Ensure that the meeting details are copied over when a person is edited.
+     */
+    @Test
+    public void execute_copyMeetingDetails_success() {
+        Person personWithMeeting = model.getFilteredPersonList().get(INDEX_THIRD_PERSON.getZeroBased());
+        PersonBuilder personInList = new PersonBuilder(personWithMeeting);
+        Person editedPerson = personInList.withName(VALID_NAME_BOB).build();
```

Comment:
Should the `editedPerson` be given a meeting value using `.withMeeting()` instead? Since this test is testing for meeting details specifically.

Reply:
`personWithMeeting` has a non-empty meeting object, so `.wtihMeeting()` is not needed.

`.withName()` is called to so in the future if we optimize `EditCommand` to do nothing if there's effectively no change, this test would still be fine